### ISSUE TITLE
Field Inputs: Input Search

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#697]
**Issue:**
1. Style of input is being read by SR (round, square) which is unnecessary.

2. SR is redundant and repeats 'search' three times when focused on the text input area. It also reads 'button' twice.

3. Does not have focus indicator.

4. Search icon button has a small target.

5. For the disabled state, SR only reads the search icon button.

**Solution:**
1. Remove the style from being read by SR.

2. Reduce redundancy - SR should only read 'search' and 'button' once.

3. Add focus indicator.

4. Increase target size for search icon button.

5. For the disabled state, SR should NOT read the search icon button. SR should either read that this input is disabled or skip it entirely (pending decision on how to treat disabled states).
